src/delegation/EscrowIVotesAdapter.sol

```diff
@@ -96,6 +96,27 @@ contract EscrowIVotesAdapter is
         emit AutoDelegationSet(sender, _disabled);
     }
 
+    function _setDelegated(uint256 tokenId, bool value) internal {
+        uint256 bucket = tokenId >> 8; // tokenId / 256
```

Comment:
I found it weird that we were using bitwise operators then immediately leaving the comment that describes the operation in more readable terms. I learned though that the SHR opocde is marginally cheaper than the DIV opcode so I guess TIL